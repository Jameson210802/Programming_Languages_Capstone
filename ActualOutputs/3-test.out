### FILE: 3-test.in
### TOKENS (stdout)

[1;33m===== BEGIN TOKENIZE =====[0m

1 PROGRAM
1 UNKNOWN

### PARSE (stdout)

[1;35m===== BEGIN PARSING =====[0m


### STDERR (tokens + parse)
[tokens stderr]
Lexical error near: ' '

[parse stderr]
Parse error: expected IDENT after PROGRAM

### EXITCODES
tokens_rc=2
parse_rc=2
