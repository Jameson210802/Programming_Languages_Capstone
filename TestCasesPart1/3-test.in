PROGRAM BaNANA;
BEGIN
  WRITE('Split happens')
END