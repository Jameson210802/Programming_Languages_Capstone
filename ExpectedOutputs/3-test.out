### FILE: 3-test.in
### TOKENS (stdout)

[1;33m===== BEGIN TOKENIZE =====[0m

1 PROGRAM
1 IDENT B
1 UNKNOWN

### PARSE (stdout)

[1;35m===== BEGIN PARSING =====[0m


### STDERR (tokens + parse)
[tokens stderr]
Lexical error near: 'a'

[parse stderr]
Parse error (line 1): expected SEMICOLON â€” after program name, got UNKNOWN [a]

### EXITCODES
tokens_rc=2
parse_rc=2
